mlc_config.json

```diff
@@ -0,0 +1,3 @@
+{
+  "retryOn429": true
+}
```

Comment:
can we put this file in `.github/workflows`? dont want to litter the top level with too many files